packages/usa-banner/src/styles/_usa-banner.component.css

```diff
@@ -0,0 +1,91 @@
+:host {
+  --icon-lock: url("../img/lock.svg");
```

Comment:
So this would expect external resources to exist in the parent/image directory? It seems a little opinionated, no?

I only ask because I remember you saying something about base64 encoding the images. Is there a way to set an asset path prefix with css custom props?